src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.html

```diff
@@ -76,7 +76,6 @@
                   class="kuiLink"
                   href="https://en.wikipedia.org/wiki/ISO_week_date"
                   target="_blank"
-                  title="Wikipedia: ISO Week Date"
```

Comment:
Never mind, I found your link in the issue for this PR: https://www.paciellogroup.com/blog/2012/01/html5-accessibility-chops-title-attribute-use-and-abuse/. I'm do think we should add this guideline to our HTML style guide (as part of a separate PR, if you prefer).